extensions/sql-database-projects/src/models/project.ts

```diff
@@ -617,6 +617,49 @@ export class Project implements ISqlProject {
 		await this.createCleanFileNode(beforeBuildNode);
 	}
 
+	public async convertProjectToSdkStyle(): Promise<void> {
+		// don't do anything if the project is already SDK style or it's an SSDT project that hasn't been updated to build in ADS
+		if (this.isSdkStyleProject || !this._importedTargets.includes(constants.NetCoreTargets)) {
+			return;
+		}
+
+		// make backup copy of project
+		await fs.copyFile(this._projectFilePath, this._projectFilePath + '_backup');
+
+		// remove SSDT and ADS SqlTargets imports
+		const importsToRemove = [];
+		for (let i = 0; i < this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.Import).length; i++) {
+			const importTarget = this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.Import)[i];
+
+			const condition = importTarget.getAttribute(constants.Condition);
+			const projectAttributeVal = importTarget.getAttribute(constants.Project);
+
+			if (condition === constants.NetCoreCondition && projectAttributeVal === constants.NetCoreTargets
```

Comment:
Is checking the condition necessary? Users could have modified those manually

Reply:
oh that's true. I'll update it to just check for the matching project attribute values